src/components/3d_editor/advanced_geometry/python_transformation/PythonCodeDisplay.tsx

```diff
@@ -0,0 +1,55 @@
+import CodeMirror from "@exabyte-io/cove.js/dist/other/codemirror";
+import theme from "@exabyte-io/cove.js/dist/theme";
+import Box from "@mui/material/Box";
+import React from "react";
+
+interface PythonCodeDisplayProps {
+    pythonCode: string;
+    pythonOutput: string;
+    setPythonCode: (pythonCode: string) => void;
+}
+
+const PythonCodeDisplay = (props: PythonCodeDisplayProps) => {
+    const { pythonCode, pythonOutput, setPythonCode } = props;
+
+    return (
+        <>
+            <Box
+                id="python-code-input"
+                sx={{
+                    border: `1px solid ${theme.palette.secondary.light}`,
+                }}
+            >
+                {/* @ts-ignore */}
```

Comment:
Why do we need to ignore here and below?

Reply:
ESlint gives this error: 
```
TS2786:  CodeMirror  cannot be used as a JSX component.
Its type  typeof CodeMirror  is not a valid JSX element type.
Types of construct signatures are incompatible.
Type 'new (props: CodeMirrorProps) => CodeMirror' is not assignable to type 'new (props: any, deprecatedLegacyContext?: any) => Component<any, any, any>'.
Type  CodeMirror  is missing the following properties from type  Component<any, any, any> :  context, setState, forceUpdate, props , and  2  more.
```
I just decided to deal with types all at once sometime later. 
This needs to be adjusted in Cove.js.